Samples/BlazorExample/BlazorExample/Client/Pages/EditPerson.razor

```diff
@@ -62,10 +62,20 @@ else
   [Parameter]
   public string id { get; set; }
 
-  protected override async Task OnInitializedAsync()
+  protected override void OnInitialized()
   {
     vm.Saved += () => NavigationManager.NavigateTo("listpersons");
+     vm.ModelChanging += (o, n) =>
```

Comment:
As this exists for teaching other, less experienced users, might it be an aid to understanding to use the names old and new for the parameters here?